### ISSUE TITLE
Can I run TreasuryCoin locally now?

### ISSUE THREAD

**Issue by liuchengxu** [input-output-hk/TreasuryCoin#7]
As mentioned in README, [treasury-crypto](https://github.com/input-output-hk/treasury-crypto) is private for now. So, is it possible to run TreasuryCoin locally at present? Thanks!

**Comment by dkaidalov** [comment]
@liuchengxu No, treasury-crypto library hasn't been published yet, so you will not be able to build the TreasuryCoin

**Comment by liuchengxu** [comment]
Thanks for your response. Then is there any chance you could provide the compiled target/jar for these interested in TreasuryCoin to  try it out?

**Comment by dkaidalov** [comment]
I will figure it out and answer a bit later
